docs/snippets/azure/AppHost/Program.ConfigurePostgresSQLInfra.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Azure.Provisioning.PostgreSql;
+
+internal static partial class Program
+{
+    public static void ConfigurePostgreSQLInfra(IDistributedApplicationBuilder builder)
+    {
+        // <configure>
+        builder.AddAzureCosmosDB("cosmos-db")
+            .ConfigureInfrastructure(infra =>
+            {
+                var flexibleServer = infra.GetProvisionableResources()
+                                          .OfType<PostgreSqlFlexibleServer>()
+                                          .Single();
+
+                flexibleServer.Sku = new PostgreSqlFlexibleServerSku
+                {
+                    Tier = PostgreSqlFlexibleServerSkuTier.Burstable,
+                };
+                flexibleServer.HighAvailability = new PostgreSqlFlexibleServerHighAvailability
+                {
+                    Mode = PostgreSqlFlexibleServerHighAvailabilityMode.ZoneRedundant,
+                    StandbyAvailabilityZone = "2",
+                };
+                flexibleServer.Tags.Add("ExampleKey", "Example value");
```

Comment:
```suggestion
```

I don't think adding Tags is that valuable to have in the docs.

Reply:
I've been including it for all provisioning examples, as I believe it is valuable. Lots of devs do tag resources often.